### DISCUSSION TITLE
Can not change the icon of links

### DISCUSSION THREAD

**Discussion by xBlueDragonn** [exoplatform/exo-community#71] [Q&A]
when i try to change the icon of a links like below: i got a preview with the icon but when i apply changes the icon do not appear
![image](https://github.com/exoplatform/exo-community/assets/145013620/05b023e0-ebc4-4f9d-a6ab-b76535ee0054)
![image](https://github.com/exoplatform/exo-community/assets/145013620/32cf1107-7dc8-4990-82ab-a218ccb4554b)
i try with jpeg png svg format

**Comment by rdenarie** [comment]
Hi
Can you create an account on https://community.exoplatform.com/
Then you can post your question in eXo Community Space, you'll have more chance to get an anwser.

Thanks
